wraith/configs/spider.yaml

```diff
@@ -0,0 +1,60 @@
+##############################################################
```

Comment:
Do we need these example ones? Out of interest what is brought in by the gem, do we need to include all this stuff in the repo?